src/logic/hooks/useEstimateSafeCreationGas.tsx

```diff
@@ -29,14 +30,15 @@ const estimateGas = async (
   safeCreationSalt: number,
   addresses: string[],
 ): Promise<SafeCreationEstimationResult> => {
-  const [gasEstimation, gasPrice, feesPerGas] = await Promise.all([
+  const [gasEstimation, gasPrice] = await Promise.all([
     estimateGasForDeployingSafe(addresses, numOwners, userAccount, safeCreationSalt),
     calculateGasPrice(),
-    getFeesPerGas(),
   ])
 
   const estimatedGasCosts = gasEstimation * parseInt(gasPrice, 10)
-  const maxPrioFeePerGas = setMaxPrioFeePerGas(feesPerGas.maxPriorityFeePerGas, parseInt(gasPrice, 10))
+  const maxPrioFeePerGas = isMaxFeeParam()
```

Comment:
Might be better to spread this into the `Promise.all` array conditionall instead of here?

Reply:
Agreed, it looks more readable. Changed it back